### ISSUE TITLE
Copyright?

### ISSUE THREAD

**Issue by eebop** [endless-sky/endless-sky-assets#179]
None of the assets here have copyright, should there be a copyright page?
If there is one it should be made easier to find (I can't find it)

**Comment by Saugia** [comment]
The copyright for all assets *is* the copyright file in the game files; we could add something akin to a page on Github, but I'm not sure what it would be other than just copy-pasting the copyright file onto Github, which you can already look at through the game's repo.

**Comment by eebop** [comment]
Yeah but blender assets are different from the pngs. maybe not to big of a deal if they're all going under the same license

**Comment by TheGiraffe3** [comment]
The only assets with a copyright not included in vanilla would be the `a template` and `z template` (see #139)
